CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 1.1.6 (Feb 21, 2023)
+
+Bug Fixes
+
+- Bluetooth permission now  also check device version in case of newer target version than device
```

Comment:
suggestion: `Bluetooth permissions now checks for the device version in case the target version is newer`?

Reply:
updated